geomesa-hbase/geomesa-hbase-tools/bin/bootstrap-geomesa-hbase-aws.sh

```diff
@@ -67,7 +67,8 @@ echo "# Auto-registration for geomesa coprocessors ${NL}export CUSTOM_JAVA_OPTS=
 
 # Deploy the GeoMesa HBase distributed runtime to the HBase root directory
 if [[ "$ROOTDIR" = s3* ]]; then
-  aws s3 cp /opt/geomesa/dist/hbase/$DISTRIBUTED_JAR_NAME ${ROOTDIR}/lib/ && \
+  availabilityZone=$(wget -q -O - http://169.254.169.254/latest/meta-data/placement/availability-zone) # Static metadata service
```

Comment:
do you know if this would work in different aws environments? would it be feasible to use the aws command instead?https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-availability-zones.html

Reply:
This works in public and gov cloud environments. I initially tried to use the aws cli but it doesn't work in gov cloud, I believe because you need to specify the region, chicken and the egg problem.

Reply:
works for me